x-pack/plugins/security_solution/public/overview/components/overview_cti_links/cti_no_events.tsx

```diff
@@ -23,7 +23,7 @@ const warning = (
 
 export const CtiNoEventsComponent = ({ to, from }: { to: string; from: string }) => {
   const { buttonHref, listItems, isDashboardPluginDisabled } = useCtiDashboardLinks(
-    { ...emptyEventCountsByDataset },
+    emptyEventCountsByDataset,
```

Comment:
Was this the cause of many rerenders, or just an unnecessary spread you also noticed? Hard to tell from the PR.

Reply:
I have a hard time reproducing so I wasn't sure if this one was the cause or the other one (non-empty eventCountsByDataset) but I think it's a good idea to ensure that the first arg of useCtiDashboardLinks is identical in this case, and compared via lodash in the case of cti_with_events.tsx